public/helpers/update-users.js

```diff
@@ -0,0 +1,35 @@
+const updatePatricipants = (sender, participants) => {
+  participants.forEach((participant) => {
+    if(participant.permissions.includes('AV') && participant.permissions.includes('CHAT')){
```

Comment:
This logic actually doesn't quite work - my bad!
(During this sprint) a mentor doesn't begin with CHAT and AV permissions by default any more, so when they first join the room, they would drop into the `else if` block and appear as a student.